### DISCUSSION TITLE
What is the .pipelines folder for?

### DISCUSSION THREAD

**Discussion by powershellwhizz** [Azure/AzOps#491] [Q&A] [answered]
I was looking thorugh the code an noticed that you have .github/workflows which from how I understand it, is where the actions and pipelines are defined, so I was confused to see the same files under a .piplines folder too. When I search Github for this folder nothing comes up, is it redundant?

thanks,

**Comment by daltondhcp** [comment, marked as answer]
In .pipelines, we have the starter pipelines for Azure Devops (https://github.com/azure/azops/wiki/azure-pipelines). So nothing to do with GitHub actions :)

**Reply by powershellwhizz** [reply]
Ahhh! Makese sense now! I completely forgot about Azure Devops. Thanks again Johan.
